onboarding/src/Components/Employee/EmployeeFormPages/EmployeeAnswerRow.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const EmployeeAnswerRow = ({ id, data, type, name, toggleChecked }) => {
+  return (
+    <tr>
+      <td className="d-flex justify-content-between align-items-center pr-0">
+        <div className="w-100">
+          <div
+            className={`w-100 custom-control custom-control-inline custom-${
+              type === "osa" ? "radio" : "checkbox"
+            }`}
+          >
+            <input
+              className={"custom-control-input"}
+              id={id}
+              name={name}
+              type={type === "osa" ? "radio" : "checkbox"}
+              value={data.title}
```

Comment:
Another reason for the error we were facing ;-)
You called it `it's not a bug, it's a feature` ;-)
and we know what to do and how to work on it (**)

Reply:
It's just a matter of the `JSON` data we had been talking about.
We agreed that data from the back-end API has to be converted from simple `string` into `JSON`.
But before this conversion, this error had occurred.

(Same there where you asked for https://github.com/binawork/OnlineOnboarding/pull/139/files?file-filters%5B%5D=.js&hide-deleted-files=true#r542306158)

Reply:
Just to remember them
`Warning: Failed prop type: Invalid prop `data` of type `string` supplied to `EmployeeAnswerRow`, expected `object`.`